apps/full-site-editing/full-site-editing-plugin/common/index.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * File for various functionality which needs to be added to Simple and Atomic
+ * sites. The code in this file is always loaded.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE\Common;
+
+/**
+ * Can be used to determine if the current screen is the block editor.
+ *
+ * @return bool True if the current screen is a block editor screen. False otherwise.
+ */
+function is_block_editor_screen() {
+	return is_callable( 'get_current_screen' ) && get_current_screen() && get_current_screen()->is_block_editor();
+}
+
+/**
+ * Adds custom classes to the admin body classes.
+ *
+ * @param string $classes Classes for the body element.
+ * @return string
+ */
+function admin_body_classes( $classes ) {
+	global $post;
+	$hide_homepage_title = (bool) get_theme_mod( 'hide_front_page_title', false );
+	$is_homepage         = ( (int) get_option( 'page_on_front' ) === $post->ID );
```

Comment:
`$post` is not always available, like outside the Editor. Is returning early an option when `is_block_editor_screen()` returns false?

Reply:
@obenland this bug should be fixed in master: https://github.com/Automattic/wp-calypso/blob/e6dd1eb093b6b6e772362b97982d3edf34ae9b9a/apps/full-site-editing/full-site-editing-plugin/common/index.php#L30-L32

Reply:
caught it after this PR was merged, unfortunately